platform/stm32f0xx/exti.c

```diff
@@ -154,11 +154,11 @@ void stm32_setup_ext_interrupt(int interrupt, stm32_ext_interrupt_port_t port,
                                bool rising_edge, bool falling_edge) {
     assert(0 <= interrupt && interrupt <= 15);
 
```

Comment:
stm32_rcc_set_enable(STM32_RCC_CLK_SYSCFGCOMP, true) is also needed here in case the peripheral is not already enabled.